server/src/packages/rwcds-form/rwcds-form.service.ts

```diff
@@ -172,13 +172,26 @@ export class RwcdsFormService {
 
       // Lookup: dcp_affectedzoningresolutions asks for a ZR type,
       const { label, code } = ZONING_RESOLUTION_TYPES.filter((zr) => zr.label === projectActionLabel)[0] || { label: '', code: null };
+      const matchingZr = affectedZoningResolutions.find(zr => zr.dcp_zoningresolutiontype === code);
```

Comment:
interesting... so there is no way to find the `matchingZr` by directly comparing `zr.dcp_zoningresolutiontype` to an attribute on `projectAction`? We have to use the `code` property in our own ZONING_RESOLUTION_TYPES lookup?

Reply:
I don't think dcp_projectaction entity has a field that has the corresponding codes, only a field that has labels (`ZC`, `ZA`)

Reply:
ahh ok!